detection_rules/schemas/__init__.py

```diff
@@ -269,8 +268,8 @@ def get_stack_versions(drop_patch=False) -> List[str]:
         return versions
 
 
-def get_min_supported_stack_version() -> Version:
+def get_min_supported_stack_version(drop_patch=False) -> Version:
     """Get the minimum defined and supported stack version."""
     stack_map = load_stack_schema_map()
     min_version = min(Version(v) for v in list(stack_map))
-    return min_version
+    return Version(min_version[:2]) if drop_patch else min_version
```

Comment:
Why the `:2` ?

Reply:
`drop_patch` is meant to drop the patch version (and beyond), so it only keeps the first 2 items in the tuple (`Version`). But this is only if `drop_patch` is set to `True`